doc/cascadia/Cascading-Default-Settings.md

```diff
@@ -444,6 +488,39 @@ the `profiles.json` file. Fortunately though, users should be able to remove
 much of the boilerplate from their `profiles.json` files, and trim it down just
 to their modifications.
 
+### Dynamic Profile Generators Need to be Enabled
+With the current proposal, profiles that are generated by a dynamic profile
+generator _need_ that generator to be enabled for the profile to appear in the

```

Comment:
This is required for things like "I uninstalled my WSL distribution, why is it still hanging around" to work, anyway. Maybe if we bang on the language a bit to change "generator" to "source"? A "profile source" can be disabled, and _that will remove all of the profiles that originated from that source._

Reply:
I think the discussion around "profile generators can be slow" should be resolved by "don't do that, you are a bad plugin author" :smile:

Reply:
This also opens us up to "default.json parser is just another type-of-source", which might be a safe mental model?